packages/flutter/test/widgets/media_query_test.dart

```diff
@@ -564,4 +564,159 @@ void main() {
     expect(outsideBoldTextOverride, false);
     expect(insideBoldTextOverride, true);
   });
+
+  String generateErrorMessage(String parameterName) {
+    return '\'$parameterName != null\': is not true';
+  }
+
+  test('size parameter in MediaQueryData cannot be null', () {
+    try {
+      MediaQueryData(size: null);
+    } on AssertionError catch (error) {
+      expect(error.toString(), contains(generateErrorMessage('size')));
```

Comment:
Maybe modify this line to:
```suggestion
      expect(error.toString(), contains('size != null') && contains('is not true'));
```

Reply:
I guess this will work, but will it be the correct way?

Reply:
It doesn't have to be the exact error message, as long as key parts of the message exist.

Reply:
Okay, I'll push the changes soon.

Reply:
`contains('size != null') && contains('is not true')` won't be valid in expect. I'll write two separate expect

Reply:
@shihaohong  It won't work because it requires a new line character.

Reply:
Can we do like this 

```dart    
try {
      MediaQueryData(size: null);
    } on AssertionError catch (error) {
       return
}
```

If it throws AssertionError, we will return otherwise the test will fail.

Reply:
Sorry, I thought I had responded to this. How come the newline character prevents the previous suggestion from working?

Reply:
I guess it was my fault, I was still checking for `'size != null'` instead of `size != null`.  The recent commit should pass the test.

Reply:
No problem! At least it's not a confusing issue :) This kind of thing happens to me all the time

Reply:
Thank you for the help 😄 @shihaohong

Reply:
It did pass all the test.